brainglobe_utils/qtpy/logo.py

```diff
@@ -51,9 +88,21 @@ def header_widget(
         The name of the package, e.g. "brainrender-napari"
     package_tagline : str
         The tagline for the package
-    tutorial_file_name : str
+    tutorial_file_name : str, optional
         The name of the tutorial file (must include .html),
-         e.g. "brainrender-qtpy.html": str
+         e.g. "brainrender-qtpy.html"
+    documentation_path : str, optional
+        The relative path of a documentation file (must include .html),
```

Comment:
> The relative path of a documentation file (must include .html),

I'd include what this is relative to (it looks like it's relative to `https://brainglobe.info/documentation/` from inspection of the `_docs_links_widget`) in the docstring itself.